src/components/pages/group/common/custom-field/index.tsx

```diff
@@ -38,21 +41,30 @@ export default function GroupCustomField({ userGroupId, groupCustomField, allCus
         </AccordionButton>
 
         <AccordionPanel p={4}>
-          {isCreate ? (
+        {isCreate && !isEdit? (
             <AddCustomFieldForm
               customFields={customFields}
               allCustomFields={customFieldList}
               setCustomFields={setCustomFields}
               setIsCreate={setIsCreate}
             />
-          ) : (
+          ) :!isEdit?(
             <CustomFieldTable
               userGroupId={userGroupId}
               customFields={customFields}
               setCustomFields={setCustomFields}
               setIsCreate={setIsCreate}
+              setIsEdit={setIsEdit}
+              setEditCustomFieldData = {setEditCustomFieldData}
             />
-          )}
+          ):(
+            <EditCustomField
+              editCustomFieldData = {editCustomFieldData}
+              setIsEdit={setIsEdit}
+              setCustomFields={setCustomFields}
+            />
+          )
```

Comment:
this blocks can be easily reversed and we can change `!isEdit` to `isEdit`, also if possible please simply both turnary conditions